var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -247,6 +257,7 @@ def test(self):
             print('Skipping: HYPRE must be installed with +mpi to run tests')
             return
 
+        print(self._cached_tests_work_dir)
```

Comment:
Is this a debug print?

Reply:
That seems to be a debug print. I don't recall if I added it, or someone else. I think that line can be simply removed. 

@balay Could you take a look at the patch files for hypre I submitted in this PR? 

var/spack/repos/builtin/packages/hypre/ij_gptune.patch
var/spack/repos/builtin/packages/hypre/package.py

It's essentially added a few lines in ij.c to print out the runtime into the runlog, and get the input parameters from command line. Thanks!

Reply:
@alalazo I have removed this debug printing line.